angular.js

```diff
@@ -32,5 +32,28 @@ module.exports = {
             Overrides to Angular extra recommended rules:
             https://github.com/angular-eslint/angular-eslint/blob/master/packages/eslint-plugin/src/configs/recommended--extra.json
         */
+
+        /*
+            Overrides to Angular rules outside of the recommended configuration:
+        */
+
+        '@angular-eslint/component-max-inline-declarations': 'error',
+        '@angular-eslint/component-selector': 'error',
+        '@angular-eslint/contextual-decorator': 'error',
+        '@angular-eslint/directive-selector': 'error',
+        '@angular-eslint/no-attribute-decorator': 'error',
+        '@angular-eslint/no-forward-ref': 'error',
+        '@angular-eslint/no-input-prefix': 'error',
```

Comment:
[@angular-eslint/no-input-prefix](http://codelyzer.com/rules/no-input-prefix)

Reply:
I avoid verbs when naming @Inputs, but we don't have a general convention to apply this rule with.

**Proposal**
Leave this rule disabled.

Reply:
@mure this thread is awaiting your vote

Reply:
@TrevorKarjanis this thread is ready to close